eovpn/openvpn3_backend/dbus.py

```diff
@@ -41,35 +49,36 @@ def watch(self, callback):
                            callback)
     
     def sub_callback(self, connection, sender_name, object_path, interface_name, signal_name, parameters, update_callback):
-    
+        
+        # https://github.com/OpenVPN/openvpn3-linux/blob/master/src/dbus/constants.hpp
         status = GLib.Variant("(uus)", parameters)
         major = status.get_child_value(0).get_uint32()
         minor = status.get_child_value(1).get_uint32()
         reason = status.get_child_value(2).get_string()
 
         logger.debug("{} {} {}".format(major, minor, reason))
 
-        if(major == 2 and minor == 4):
+        if(major == OVPN3Constants.StatusMajor.CONNECTION.value and minor == OVPN3Constants.StatusMinor.CFG_REQUIRE_USER.value):
```

Comment:
I believe you can here use this construct:
```python
OVPN3Constants.StatusMajor(major) == OVPN3Constants.StatusMajor.CONNECTION`
```
That might be easier to read.

Reply:
Maybe even better to do it higher up ....

```python
major = OVPN3Constants.StatusMajor(status.get_child_value(0).get_uint32())
```

Reply:
This is def better 👍

Reply:
done 5db54044397a2b01f42650fe2e430c90a0da9786 :tada: